From c31b8512f4e20c1adeaa90b39d899a71fcbd793e Mon Sep 17 00:00:00 2001
From: Daron Chabot <chabot@frib.msu.edu>
Date: Fri, 24 Mar 2017 15:38:12 -0400
Subject: [PATCH 2/2] BLD: account for libasyn dependency

---
 pmacCoordApp/src/Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/pmacCoordApp/src/Makefile b/pmacCoordApp/src/Makefile
index 030b5df..b57601a 100644
--- a/pmacCoordApp/src/Makefile
+++ b/pmacCoordApp/src/Makefile
@@ -2,13 +2,9 @@ TOP=../..
 include $(TOP)/configure/CONFIG
 
 DBD += pmacAsynCoord.dbd
-DBD += pmacCoord_vdct.dbd
-pmacCoord_vdct_DBD += base.dbd
-pmacCoord_vdct_DBD += asyn.dbd
-pmacCoord_vdct_DBD += stream.dbd
-pmacCoord_vdct_DBD += calcSupport.dbd
 
 LIBRARY = pmacAsynCoord
+pmacAsynCoord_LIBS = asyn
 pmacAsynCoord_SRCS = pmacAsynCoord.c pmacAsynCoordRegister.cc
 
 include $(TOP)/configure/RULES
-- 
2.1.4

